hack/test/e2e-qemu.sh

```diff
@@ -63,6 +63,23 @@ case "${WITH_CONTROL_PLANE_PORT:-false}" in
     ;;
 esac
 
+case "${VIA_MAINTENANCE_MODE:-false}" in
+  false)
+    ;;
+  *)
+    # apply config via maintenance mode
+    QEMU_FLAGS="${QEMU_FLAGS} --disable-injecting-config --with-apply-config"
```

Comment:
e2e test seems to be failing because of unknown parameter --disable-injecting-config.

Reply:
yep, thanks, fix is coming :)